lib/elixir_analyzer/exercise_test/assert_call.ex

```diff
@@ -133,14 +133,16 @@ defmodule ElixirAnalyzer.ExerciseTest.AssertCall do
         nil ->
           nil
 
+        # For erlang libraries: import :math
+        module when is_atom(module) ->
+          module
+
         {:__aliases__, _, module} ->
           module
 
         x ->
           raise ArgumentError,
-                "calling function signature requires :module to be nil or a module atom, got: #{
-                  inspect(x)
-                }"
+                "calling function signature requires :module to be nil or a module atom, got: #{inspect(x)}"
```

Comment:
(This is general Elixir dev feedback, not related to the feature)

This formatting change tells me that you're using Elixir 1.12, which in turn tells me you're not using asdf (https://asdf-vm.com/#/) to manage Elixir versions. I know this because if you were using asdf, it would automatically read `.tool-versions` and switch to the appropriate version, and there was a change in how strings get formatted between 1.11 and 1.12 😁 

We forgot to update this repository to Elixir 1.12 when we updated the other ones 🙈. You can see in the file `.tool-versions` (and also by the Docker container used in the Github Action).

Anyway, you don't need to change anything. I'm going to open PRs that updates Elixir here to 1.12 and that does the formatting check on CI to catch this automatically.

Reply:
Elementary, my dear Watson. Nice deductions :)

I don't know asdf, I'm looking at the webpage. Do you recommend it?

Reply:
If you are already using homebrew or something to manage versions, no.  If you aren't using anything and are installing them directly on to your machine, yes.

Reply:
asdf is excellent for synchronizing dependency versions in teams, especially when there's more than one to manage (Erlang, Elixir, NodeJS). I definitely recommend it if you're working on different projects that require different versions of the same language. If not, then it's a nice to have but won't be gamechanging.

I have bad experiences with `homebrew`. Basically any time I install any package, it updates random unrelated packages which sometimes leads to things breaking unexpectedly. For example installing PHP broke my Postgres installation once 🤷

Reply:
Yeah, I think homebrew broke my Postgres too once. I see the need for asdf and I will probably try it out at some point.